src/app/organizations/collection/collection.component.ts

```diff
@@ -69,6 +70,7 @@ export class CollectionComponent implements OnInit {
     public dialog: MatDialog,
     private activatedRoute: ActivatedRoute,
     private userQuery: UserQuery,
+    private orgLogoService: OrgLogoService,
```

Comment:
This feels like it should be public, because you access it from the template. I was surprised this built at all, but it looks like its enforcement depends on settings that I didn't bother to figure out.